packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6224,6 +6246,18 @@ export default class Meeting extends StatelessWebexPlugin {
               stack: error.stack,
             });
 
+            // @ts-ignore
+            this.webex.internal.newMetrics.submitClientEvent({
+              name: 'client.share.floor-granted.local',
```

Comment:
before sending the request we send "client.share.floor-grant.request" and here when it failed we send "client.share.floor-granted.local" - is that the right event? shouldn't it be "client.share.floor-grant.request" or something like "client.share.floor-grant.response"?

Reply:
I was told that the result of this request should send `client.share.floor-granted.local`. This can either pass, or in the catch block, fail.

Reply:
@johnsoter13 , told by Meera?

Reply:
Yes

Reply:
ok, sounds good. For the success case there is already existing code that sends `client.share.floor-granted.local` - see setUpLocusMediaSharesListener(). This place here in the catch looks good for that event. Not sure if we need to send the event with some error also in the case if changeMeetingFloor() succeeds but later we don't get the floor granted confirmation via Mercury

Reply:
For the case where we don't get confirmation from Mercury, we can handle that case in a later PR.

Reply:
Open to discussion on this!